CONTRIBUTING.md

```diff
@@ -0,0 +1,272 @@
+
+   ```bash
+   git checkout -b develop
+   git pull upstream develop
+   ```
+
+3. Initiate git flow in the repository and set up a feature branch.
+
+   ```bash
+   git flow init
+   Which branch should be used for bringing forth production releases?
+      - develop
+      - master
+   Branch name for production releases: [master] master
+   
+   Which branch should be used for integration of the "next release"?
+      - develop
+      - master
+   Branch name for "next release" development: [develop] develop
+   
+   How to name your supporting branch prefixes?
+   Feature branches? [feature/] issue-
+   Bugfix branches? [bugfix/] issue-
+   Release branches? [release/] issue-
+   Hotfix branches? [hotfix/] issue-
+   Support branches? [support/] issue-
+   Version tag prefix? []
+   ```
+   
+   ```bash
+   git flow feature start <issue number>-<feature name>-<github username>
+   ```
+
+4. Commit your changes in logical chunks. Please adhere to these [git commit
+   message guidelines](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html)
+   or your code is unlikely be merged into the main project. Use Git's
+   [interactive rebase](https://help.github.com/articles/interactive-rebase)
+   feature to tidy up your commits before making them public.
+   
+   We advise including the issue number in the commit message as well.
+
+5. Locally merge the upstream development branch into your topic branch:
+
+   ```bash
+   git pull upstream develop
+   ```
+
+6. Push your topic branch up to your fork:
+
+   ```bash
+   git push origin issue-<issue number>-<topic-branch-name>-<github-username>
+   ```
+
+7. [Open a Pull Request](https://help.github.com/articles/using-pull-requests/) against the `develop` branch.
+    - Add a clear title and description
+    - Include screenshots of the before and after if your changes include differences in HTML/CSS. Drag and drop the images into the body of your pull request.
+    - Reference any relevant issues or supporting documentation in your PR (ex. “Closes #37.”)
+    - Make sure submitted code has no conflicts
+    - Review your PR code to ensure there is no extra edits than required to resolve the issue. For example no additional refactoring, no extra lines of code, no irrelavent bug fixes.
+
+**IMPORTANT**: By submitting a patch, you agree to allow the project owners to
+license your work under the terms of the [MPL-2.0 License](https://github.com/blockstack/blockstack-browser/blob/master/LICENSE.md) (if it
+includes code changes) and under the terms of the
+[Creative Commons Attribution 3.0 Unported License](docs/LICENSE.md)
+(if it includes documentation changes).
+
+[^ Back To The Top](#contributing-to-blockstackorg)
+
+## Code guidelines
+
+### HTML
+
+[Adhere to the Code Guide.](http://codeguide.co/#html)
+
+- Use tags and elements appropriate for an HTML5 doctype (e.g., self-closing tags).
+
+### JS
+
+- No semicolons (in client-side JS)
```

Comment:
Let's remove this and define the javascript requirements programmatically with the `.eslintrc` file instead of english.